File: 4/MainActivity.java
package com.jfdimarzio.homework4by2784780;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {
    Button Linear,Table,Relative;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Linear = (Button) findViewById(R.id.Linearlayout);
        Relative = (Button) findViewById(R.id.Relativelay);
        Table = (Button) findViewById(R.id.Tablelayout);
        Linear.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, LinearLayout.class);
                startActivity(intent);
            }
        });
        Relative.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, RelativeLayout.class);
                startActivity(intent);
            }
        });
        Table.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, TableLayout.class);
                startActivity(intent);
            }
        });
    }

}

File: 4/MainActivity.kt
package com.jfdimarzio.homework4by2784780

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    lateinit var linear: Button
    lateinit var table: Button
    lateinit var relative: Button

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        linear = findViewById(R.id.Linearlayout)
        relative = findViewById(R.id.Relativelay)
        table = findViewById(R.id.Tablelayout)

        linear.setOnClickListener {
            val intent = Intent(this@MainActivity, LinearLayout::class.java)
            startActivity(intent)
        }

        relative.setOnClickListener {
            val intent = Intent(this@MainActivity, RelativeLayout::class.java)
            startActivity(intent)
        }

        table.setOnClickListener {
            val intent = Intent(this@MainActivity, TableLayout::class.java)
            startActivity(intent)
        }
    }
}
